lib/classes/Service.js

```diff
@@ -27,13 +27,13 @@ class Service {
   load(opts) {
     const that = this;
     const options = opts || {};
-    const servicePath = that.S.instances.config.servicePath;
+    const servicePath = that.serverless.config.servicePath;
 
     if (!servicePath) {
       throw new Error('ServicePath is not configured.');
     }
 
-    return that.S.instances.yamlParser
+    return that.serverless.yamlParser
```

Comment:
The YamlParser should be imported at the top (otherwise we have a dependency on the serverless instance)